src/routes/+page.server.ts

```diff
@@ -8,9 +8,12 @@ const imageModules = import.meta.glob('$lib/generated/icons/*.svg', {
 
 export function load({ request }) {
   const term = new URL(request.url).searchParams.get('q');
+  const excludeFolders = Boolean(
+    new URL(request.url).searchParams.get('excludeFolders'),
+  );
```

Comment:
So you don't create the `URL` twice:

```suggestion
  const { searchParams } = new URL(request.url);
  const term = searchParams.get('q');
  const excludeFolders = Boolean(
    searchParams.get('excludeFolders'),
  );
```